presto-native-execution/presto_cpp/main/connectors/PrestoToVeloxConnector.h

```diff
@@ -52,6 +52,9 @@ std::unique_ptr<velox::connector::ConnectorTableHandle> toHiveTableHandle(
     const VeloxExprConverter& exprConverter,
     const TypeParser& typeParser);
 
+velox::common::CompressionKind toFileCompressionKind(
```

Comment:
If you are separating a HiveToPrestoVeloxConnector file then this can be moved there as well.